packages/brick-container/src/styles/theme/var.css

```
@@ -0,0 +1,113 @@
+:root {
+  /** light palette **/
+  --light-palette-gray-1: #fff;
```

dark/light 的变量名应该一样，通过 data-theme 去设置不同的值。

```suggestion
  --palette-gray-1: #fff;
```